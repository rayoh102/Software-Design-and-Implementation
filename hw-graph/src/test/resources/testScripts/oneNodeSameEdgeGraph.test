# Tests adding one node with edge to itself to graph

# Create a graph
CreateGraph oneNodeSameEdgeGraph

# Add a node
AddNode oneNodeSameEdgeGraph a

# Add an edge
AddEdge oneNodeSameEdgeGraph a a AA

# Print the nodes in the graph and the outgoing edges from a
ListNodes oneNodeSameEdgeGraph
ListChildren oneNodeSameEdgeGraph a
