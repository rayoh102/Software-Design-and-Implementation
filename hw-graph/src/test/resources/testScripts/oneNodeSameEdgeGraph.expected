# Tests adding one node with edge to itself to graph

# Create a graph
created graph oneNodeSameEdgeGraph


# Add a node
added node a to oneNodeSameEdgeGraph


# Add an edge
added edge AA from a to a in oneNodeSameEdgeGraph


# Print the nodes in the graph and the outgoing edges from a
oneNodeSameEdgeGraph contains: a
the children of a in oneNodeSameEdgeGraph are: a(AA)